service/src/main/resources/application.yml

```diff
@@ -73,6 +76,7 @@ skills:
     enabled: false
   clientLibVersion: @maven.build.timestamp@
   config:
+    publiclyExposePrometheusMetrics: true
```

Comment:
@cjmoses - did you want to set this to true in the application.properties file packaged within the jar?  This will override the default value set to false in the PortalWebSecurityHelper class

Reply:
nope that was meant to have been removed!